packages/communication-react/review/communication-react.api.md

```diff
@@ -1213,14 +1214,12 @@ export interface JumpToNewMessageButtonProps {
 // @public
 export const lightTheme: PartialTheme & CallingTheme;
 
-// @public
-export const LocalizationProvider: (props: LocalizationProviderProps) => JSX.Element;
-
-// @public
-export type LocalizationProviderProps = {
-    locale: ComponentLocale;
+// @public (undocumented)
+export const LocalizationProvider: (props: {
```

Comment:
Let's follow the same API pattern as the adapter here:

    interface LocalizationProviderProps {
        locale?: ComponentLocale;
        loader?: () => Promise<ComponentLocale>;
    }

And add comments that `locale` is used initially, and if `loader` is provided, it is called to asynchronously load the locale to use once it becomes available.